lib/Swagger/src/Discussion/Models/UpvoteHalResponse.php

```diff
@@ -66,7 +66,7 @@ class UpvoteHalResponse implements ArrayAccess
         'upvote_count' => 'int',
         '_links' => '\Swagger\Client\Discussion\Models\HalLinks',
         'requester_id' => 'string',
-        '_embedded' => '\Swagger\Client\Discussion\Models\Embeddable[]'
+        '_embedded' => 'map[string,\Swagger\Client\Discussion\Models\Embeddable[]]'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "_embedded" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)